ldsc/src/main/resources/mergeRegions.pl

```diff
@@ -5,18 +5,19 @@
 my $s3dir="s3://dig-analysis-data/out/ldsc";
 
 # which partition is being made
-my $partition=$ARGV[0];
-my $sub_region=$ARGV[1];
+my $sub_region=$ARGV[0];
+my $partition_match=$ARGV[1];
+my $partition_output=$ARGV[2];
 # Have to figure out the sub region here to fix things
 
 # create a temporary file to download all the partition data into
 my $tmpFile="partitions.csv";
 my $sortedFile="sorted.csv";
-my $bedFile="$partition.csv";
+my $bedFile="$partition_output.csv";
 
 # mergepart files together and sort it by position
-`hadoop fs -getmerge -nl -skip-empty-file "$s3dir/regions/$sub_region/partitioned/*/partition=$partition/part-*" "$tmpFile"`;
-`sort -u -k1,1 -k2,2n "$tmpFile" > "$sortedFile"`;
+`hadoop fs -getmerge -nl -skip-empty-file "$s3dir/regions/partitioned/*/partition=$partition_match/part-*" "$tmpFile"`;
+`sort -k1,1 -k2,2n "$tmpFile" > "$sortedFile"`;
```

Comment:
Bugfix here. Eliminate the (breaking) bug of `-u` which removed some lines from the input file.